streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/security/impl/AuthenticatorImpl.java

```diff
@@ -37,10 +39,29 @@ public class AuthenticatorImpl implements Authenticator {
   @Autowired private LdapService ldapService;
 
   @Override
-  public User authenticate(String username, String password) {
+  public User authenticate(String username, String password, String loginType) throws Exception {
+
+    LoginType loginTypeEnum;
+
+    try {
+      loginTypeEnum = LoginType.valueOf(loginType);
+    } catch (IllegalArgumentException e) {
+      throw new ApiAlertException(
+          String.format("the login type [%s] is not supported.", loginType));
+    }
+
+    if (loginTypeEnum.equals(LoginType.PASSWORD)) {
+      return passwordAuthenticate(username, password);
+    } else {
+      return ldapAuthenticate(username, password);
+    }
+  }
+
+  private User passwordAuthenticate(String username, String password) {
     User user = usersService.findByName(username);
-    if (user == null) {
-      return null;
+    if (user == null || user.getLoginType() != LoginType.PASSWORD) {
```

Comment:
We don't need to use this judgment：
`user.getLoginType() != LoginType.PASSWORD`